test.tf

```diff
@@ -108,24 +123,99 @@ resource "azurerm_postgresql_server" "example" {
   sku_name                     = "B_Gen5_2"
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure that vulnerability assessment is configured using SQL security policy for Azure MSSQL Server Vulnerability Assessment**
 
**Vulnerability**
Disabling Vulnerability Assessment policy for Azure MSSQL Server will remove capabilities of discovery and tracking, which may make Azure MSSQL Server open to potential database vulnerabilities.
 
**Remediation**
Make sure that Azure MSSQL Server Vulnerability Assessment is configured using SQL security policy which further is associated with Azure SQL Server.

source: accurics-beta